forum/backends/mysql/api.py

```diff
@@ -563,8 +564,8 @@ def get_sort_criteria(sort_key: str) -> list[str]:
         sort_key_mapper = {
             "date": "-created_at",
             "activity": "-last_activity_at",
-            "votes": "-votes__point",
-            "comments": "-comment_count",
+            "votes": "-votes_point",
+            "comments": "-comments_count",
```

Comment:
field is name as `comment_count`.

Reply:
This is intentional because we have a "comment_count" property in the Model. To differentiate this field from the annotated field, I have renamed it to comments_count.